alfresco-search/src/main/java/org/alfresco/solr/tracker/MetadataTracker.java

```diff
@@ -152,6 +157,19 @@ private void trackRepository() throws IOException, AuthenticationException, JSON
             checkRepoAndIndexConsistency(state);
         }
 
+        if(docRouter instanceof DBIDRangeRouter)
```

Comment:
There is an unlikely timing issue here.
Expansion is registered  as part of tracking. At start up you could expand before this is called.
I am not sure where this should live but it should get done once

Reply:
I ran into this during manual testing. I'll think about an approach for dealing with this in the short term.